Time to play:
Loading lists of data
graph data Data(x=[81, 1], edge_index=[2, 324], y=[81], pos=[81, 2])
Traceback (most recent call last):
  File "/home/dami/Inverse_GNN/apply_GS.py", line 70, in <module>
    nm = nmat[i,:,:] @ transformedgraph[i,:,:].squeeze().T
ValueError: matmul: Input operand 1 has a mismatch in its core dimension 0, with gufunc signature (n?,k),(k,m?)->(n?,m?) (size 31 is different from 81)
(81, 81)
(1, 3, 31)
(100, 3, 81)
(3, 81)
(3, 31)